src/app/pages/openstax-tutor/materials/materials.js

```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react';
+import {LabeledSection} from '../common';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import './materials.css';
+
+export default function Materials({model: {availableBooksHeader: headline, tutorBooks: books}}) {
+    const headerLabel = 'Materials';
+
+    return (
+        <LabeledSection headerLabel={headerLabel} headline={headline}>
+            <div className="materials">
+                {
+                    books.map((book) =>
+                        <div className="book-card" key={book}>
```

Comment:
not sure about preact, but react requires keys to be strings. book is an object here

Reply:
In Preact, any value can be used, though non-object-references are preferred. I should really use index for these, as they don't re-render. Really, I ought to turn off key for the line as a sort of pragma, I guess (which functionally uses index as key, without explicitly having to do so).
https://stackoverflow.com/a/42775667/392102